site/content/docs/browser-checks/screenshots.md

```diff
@@ -9,20 +9,13 @@ menu:
 You can take up to ten screenshots per browser check. This is really handy when debugging a failure situation or just
 in general to get a comfy feeling everything is OK.  
 
-Add a screenshot anywhere by using the following syntax in your script:
- 
-{{< tabs "Basic screenshot example" >}}
-{{< tab "Playwright" >}}
-```js
-await page.screenshot({ path: 'my_screenshot.png' })
-```
-{{< /tab >}}
-{{< tab "Puppeteer" >}}
+Using Playwright or Playwright Test add a screenshot anywhere by using the following syntax in your script:
+
 ```js
 await page.screenshot({ path: 'my_screenshot.png' })
 ```
-{{< /tab >}}
-{{< /tabs >}}
+
+> While screenshots are a great way to capture a specific moment of your check, we highlight recommend using it together with `@playwright/test` to record videos of your entire check run and enable step-by-step trace views.
```

Comment:
s/highlight/highly

I'd also write it's better to use traces/vidoes rather than screenshots.